openforcefields/offxml/openff_unconstrained-2.0.0.offxml

```diff
@@ -0,0 +1,369 @@
+<?xml version="1.0" encoding="utf-8"?>
+<SMIRNOFF version="0.3" aromaticity_model="OEAroModel_MDL">
+	<Author>The Open Force Field Initiative</Author>
```

Comment:
The other FFs have [flavor text](https://github.com/openforcefield/openff-forcefields/blob/db26648a777480304d3088b11653dc54b4a54d3d/openforcefields/offxml/openff-1.3.1.offxml#L3-L5) here like
```
    <!-- This is the version 1.3.1 "Parsley" force field released by the Open Force Field Initiative. -->
    <!-- The Parsley force field is available with and without bond constraints to hydrogen. -->
    <!-- This file contains bond constraints to hydrogen, and is suitable for long-timestep simulations. -->
```

Do we want to keep that up? It's not in the release guide so I could take it or leave it (and in fact it's super annoying because comments don't survive XML roundtrips so they can easily get forgotten)

Reply:
Let's drop this as a practice -- This is just a lot of manual work and it doesn't add much to the user experience.